library/src/test/java/io/ona/kujaku/layers/BoundaryLayerTest.java

```diff
@@ -157,4 +158,28 @@ public void calculateCenterPoints() {
         assertPointEquals(Point.fromLngLat(6.5d, 6.5d), point1);
         assertPointEquals(Point.fromLngLat(9.1d, 9.1d), point2);
     }
+
+    @Test
+    public void getLayerIds() {
```

Comment:
Same issue with the naming convention.

Reply:
Same as above

Reply:
Refer to https://github.com/onaio/kujaku/pull/274#discussion_r265145397.